docs/upgrade-to-v5.md

```diff
@@ -100,3 +100,7 @@ The following deprecated methods have been removed:
 - `ObjectId.prototype.get_inc`
 - `ObjectId.get_inc`
   - The `static getInc()` is private since invoking it increments the next `ObjectId` index, so invoking would impact the creation of subsequent ObjectIds.
+
+### Negative Zero is now serialized to Double
+
+While use cases for negative zero may be rare it is important that the value is preserved on round trips through the BSON serialization layer. Prior to this change negative zero was possible to preserve with the BSON `new Double(-0)` type, such code can be updated to use `-0` directly.
```

Comment:
```suggestion
4x versions of BSON incorrectly serialized the literal -0 as an Int32, which does not support -0.  This results in the following behavior: 

```typescript
deserialize(serialize({ number: -0 })); // { number: 0 }
deserialize(serialize({ number: new Double(-0) })) // { number: Double(-0) }
```

BSON now automatically serializes -0 to a Double.
```typescript
deserialize(serialize({ number: -0 })); // { number: Double(-0) }
```

Any code previously serializing -0 as a double can now use the literal `-0` syntax safely.
```

Reply:
updated, I kept it short and simple